dev/com.ibm.ws.clientcontainer.beanvalidation_fat/.classpath

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="fat/src"/>
+	<classpathentry kind="src" path="test-applications/ApacheBvalConfig.jar/src"/>
+	<classpathentry kind="src" path="test-applications/beanvalidation.jar/src"/>
+	<classpathentry kind="src" path="test-applications/BeanValidationCDI.jar/src"/>
+	<classpathentry kind="src" path="test-applications/defaultbeanvalidation.jar/src"/>
+	<classpathentry kind="src" path="test-applications/DefaultBeanValidationCDI.jar/src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/fattest.simplicity"/>
+	<classpathentry kind="lib" path="/ant_build/lib/junit.jar"/>
```

Comment:
I believe this junit.jar entry should be removed as well.  It should be part of the bnd bundle path as "/build.sharedResources/lib/junit/old/junit.jar"

Can we change all binary checkins of junit.jar to point to /build.sharedResources/lib/junit/old/junit.jar instead?
For example com.ibm.ws.clientcontainer.beanvalidation.fat.ApacheBvalConfig_11/lib/global/junit.jar